.travis.yml

```diff
@@ -164,17 +165,17 @@ script:
   - python -m pytest -v --durations=10 python/ray/tune/test/automl_searcher_test.py
 
   # ray rllib tests
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_catalog.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_filters.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_optimizers.py
-  - python -m pytest -v --durations=10 python/ray/rllib/test/test_evaluators.py
+  - python python/ray/rllib/test/test_catalog.py
+  - python python/ray/rllib/test/test_filters.py
+  - python python/ray/rllib/test/test_optimizers.py
+  - python python/ray/rllib/test/test_evaluators.py
 
   # Python3.5+ only. Otherwise we will get `SyntaxError` regardless of how we set the tester.
   - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest -v --durations=10 python/ray/experimental/test/async_test.py
 
-  - python -m pytest -v --durations=10 python/ray/test/test_global_state.py
-  - python -m pytest -v --durations=10 python/ray/test/test_queue.py
-  - python -m pytest -v --durations=10 python/ray/test/test_ray_init.py
+  - python python/ray/test/test_global_state.py
+  - python python/ray/test/test_queue.py
+  - python python/ray/test/test_ray_init.py
```

Comment:
`python -m pytest -v --durations=10 ...`

Just executing the file is a no-op I think because the test has no `__main__`.

Reply:
Ah ok, then I might need to add one, see the discussion in https://github.com/ray-project/ray/issues/4031. Or alternatively move cluster_utils to somewhere else.